akka-stream-tests/src/test/scala/akka/stream/io/FileSinkSpec.scala

```diff
@@ -116,22 +116,23 @@ class FileSinkSpec extends StreamSpec(UnboundedMailboxConfig) {
       }
     }
 
-    // FIXME: overriding dispatcher should be made available with dispatcher alias support in materializer (#17929)
     "allow overriding the dispatcher using Attributes" in assertAllStagesStopped {
-      pending
       targetFile { f ⇒
         val sys = ActorSystem("dispatcher-testing", UnboundedMailboxConfig)
         val materializer = ActorMaterializer()(sys)
         implicit val timeout = Timeout(3.seconds)
 
         try {
           Source.fromIterator(() ⇒ Iterator.continually(TestByteStrings.head))
-            .to(FileIO.toPath(f))
-            .withAttributes(ActorAttributes.dispatcher("akka.actor.default-dispatcher"))
+            .to(FileIO.toPath(f).withAttributes(ActorAttributes.dispatcher("akka.actor.default-dispatcher")).named("my-fileSink"))
+            //            .to(FileIO.toPath(f).withAttributes(ActorAttributes.dispatcher("akka.actor.default-dispatcher")))
+            //            .to(FileIO.toPath(f)).withAttributes(ActorAttributes.dispatcher("akka.actor.default-dispatcher"))
```

Comment:
@drewhk Here is something that I think is another attribute problem.
3 different cases:
L127: this works
L128: the default name "fileSink" is lost
L129: note the nesting in `to()`, the default name works, but the dispatcher attribute is not applied